packages/utils/src/build-context.js

```diff
@@ -245,12 +245,28 @@ function getAvatarUrl(hash = 'HEAD') {
   });
   if (result.status !== 0) {
     throw new Error(
-      "Unable to determine commit author's avatar URL because `git log --format=%aE -n 1` failed to provide author's email. " +
-        'This can be overridden with setting LHCI_BUILD_CONTEXT__AVATAR_URL env.'
+      'Unable to determine commit author email with `git log --format=%aE -n 1`. ' +
```

Comment:
I'm starting to think we should first fallback to `getAuthor(hash)` and try to extract the email from the `<user@example.com>` bit before falling back to git log *again*, WDYT are you willing to add that here? :D

Reply:
I had a similar thought but hesitated, mainly because introducing a regex to extract the email will be more fallible than asking git for it, and without the build context vars falling back to `getAuthor` would still call git log twice. (unless the author is stored somewhere we can pull from?)

As a compromise, `getAuthorEmail` could look for the `AUTHOR` env again, and extract from that if present, but otherwise still use git. And if the AUTHOR env var exists but doesn't match, say, `/^.* <(.+)>$/`, throw an error. How does that sound?

Reply:
cool yeah something like that sounds good, I just put up my suggestion. I don't think we want to have to manage the author vars in two places so calling it worst case 3 times so we can dedupe our management of overrides seems fine . it's not a live, super time-sensitive CLI call  🤷